src/wp-admin/about.php

```diff
@@ -53,429 +53,155 @@
 				</div>
-				<h3 class="is-smaller-heading"><?php _e( 'Improved admin and editor accessibility' ); ?></h3>
-				<p>
-					<?php
-					printf(
-						/* translators: %s: Link to accessibility improvements dev note. */
-						__( 'More than 40 improvements in accessibility include resolving focus loss problems in the editor, improving form labels and audible messages, making alternative text easier to edit, and fixing the sub-menu overlap in the expanded admin side navigation at smaller screen sizes and higher zoom levels. Learn more about <a href="%s">accessibility in WordPress</a>.' ),
-						'https://make.wordpress.org/core/2022/10/11/wordpress-6-1-accessibility-improvements/'
-					);
-					?>
-				</p>
+				<h3><?php _e( 'Focus on writing with Distraction Free mode' ); ?></h3>
+				<p><?php _e( 'For those times you want to be alone with your ideas. You can now hide all your panels and controls, leaving you free to bring your content to life.' ); ?></p>
 			</div>
 			<div class="column">
 				<div class="about__image">
-					<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false">
-						<rect width="48" height="48" rx="4" fill="#1E1E1E"/>
-						<path fill-rule="evenodd" clip-rule="evenodd" d="M23.019 25.517l-4.258 4.385-.718-.697 4.212-4.337-5.752-.025.005-1 5.826.025-3.997-4.116.717-.696 3.952 4.07-.03-5.623 1-.005.03 5.567 3.894-4.01.717.697-4.007 4.126 6.046.026-.005 1-5.942-.025 4.201 4.326-.717.697-4.174-4.298.032 6.048-1 .006-.032-6.14z" fill="#fff"/>
-					</svg>
+					<img src="data:image/svg+xml,%3Csvg width='436' height='436' viewbox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='%231d35b4' /%3E%3C/svg%3E" alt="" />
 				</div>
-				<h3 class="is-smaller-heading"><?php _e( 'Other notes of interest' ); ?></h3>
-				<p><?php _e( '6.1 includes a new time-to-read feature showing content authors the approximate time-to-read values for pages, posts, and custom post types.' ); ?></p>
-				<p>
-					<?php
-					printf(
-						/* translators: %s: "General Settings" admin page title, linked to the page if the user can edit options. */
-						__( 'The site tagline is empty by default in new sites but can be modified in %s.' ),
-						current_user_can( 'manage_options' ) ? '<a href="' . esc_url( admin_url( 'options-general.php' ) ) . '">' . __( 'General Settings' ) . '</a>' : __( 'General Settings' )
-					);
-					?>
-				</p>
-				<p><?php _e( 'A new modal design offers a background blur effect, making it easier to focus on the task at hand.' ); ?></p>
+				<h3><?php _e( 'Experience the Site Editor, now out of beta' ); ?></h3>
+				<p><?php _e( 'Stable and ready for you to dive in and explore: 6.2 is your personal invitation to discover what the next generation of WordPress—and Block themes—can do.' ); ?></p>
```

Comment:
Maybe some missing spaces here?
`WordPress — and Block themes — can do`

Reply:
well, no it looks good in en_US
https://prowritingaid.com/grammar/1000066/Should-I-use-spaces-around-an-em-dash

Reply:
Looks like without spaces is part of the WP brand style: ["Use an em dash (—) without spaces."](https://make.wordpress.org/marketing/handbook/resources/style-guide-and-brand-book/#punctuation)